pkg/strc/middleware_logger_test.go

```diff
@@ -115,3 +115,29 @@ func TestEchoRequestLoggerError(t *testing.T) {
 	assert.True(t, logHandler.Contains(int64(0), "response", "length"))
 	assert.True(t, logHandler.Contains(slog.LevelError.String(), "level"))
 }
+
+func TestEchoMiddlewareLogger(t *testing.T) {
```

Comment:
Hm what exactly is this one testing that the other tests aren't? If it's the contextSetLogger how about `TestEchoContextSetLogger` or something?